### PR TITLE
fix long text overlapping the arrow

### PR DESCRIPTION
**Pull request by gsaray**
State select dropdown in estimate shipping modal - default text is too long (on smaller screen 320px)

## Changes
- add the triangle as a background to field__input instead of select
- add fade to cover the long text

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Add an item to cart and go to cart
- Proceed to checkout and scroll to state/province

## Note 
- This can't be merged before https://github.com/mobify/progressive-web-sdk/pull/532 is merged and deployed

### CODE REVIEW

**Comment by mlworks** [comment]
Blocked until https://github.com/mobify/progressive-web-sdk/pull/532 is merged and deployed.

**Comment by mlworks** [comment]
@gsaray There's a lot of other changes in this PR, should they also be in here?  (I'm expecting to only see `select` related styling changes only)

**Comment by gsaray** [comment]
@mlworks I originally branched this off of develop and that's why it had so many other commits. since checkout-UI-fixes is updated, you only see `select` changes now.

File: web/app/styles/themes/pw-components/_field.scss [hunk 1]
```diff
@@ -4,6 +4,8 @@
 // 1. Keep the original font color of the label instead of error color - as per design
 
 $field__caution-icon: 'data:image/svg+xml;utf8,<svg width="20px" height="20px" fill="#dc0a3c" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><g stroke="none" stroke-width="1" fill-rule="evenodd"><g><path d="M10,3 L18,17 L2,17 L10,3 Z M10.7604457,13 L9.23955432,13 L9,7 L11,7 L10.7604457,13 Z M9,15 C9,14.7080853 9.09536447,14.4684428 9.28609626,14.2810651 C9.47682805,14.0936874 9.71479358,14 10,14 C10.2852064,14 10.523172,14.0936874 10.7139037,14.2810651 C10.9046355,14.4684428 11,14.7080853 11,15 C11,15.2919147 10.9046355,15.5315572 10.7139037,15.7189349 C10.523172,15.9063126 10.2852064,16 10,16 C9.71479358,16 9.47682805,15.9063126 9.28609626,15.7189349 C9.09536447,15.5315572 9,15.2919147 9,15 Z"></path></g></g></svg>';
+$base__select-triangle: 'data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="#4E439B"><polygon points="0 0 12 0 5.99940185 6"></polygon></svg>';
```
Comment (mlworks) [inline web/app/styles/themes/pw-components/_field.scss]: These variables are still in `base/_forms.scss`, I would remove those if they are not being used anymore and for these ones, I would use `field__` for naming.

**Pull request merged**